Foundation/NSJSONSerialization.swift

```diff
@@ -131,6 +130,14 @@ open class JSONSerialization : NSObject {
             }
         }
         
+        let count = jsonStr.lengthOfBytes(using: .utf8)
+        let bufferLength = count+1 // Allow space for null terminator
+        var utf8: [CChar] = Array<CChar>(repeating: 0, count: bufferLength)
+        if !jsonStr.getCString(&utf8, maxLength: bufferLength, encoding: .utf8) {
+            fatalError("Failed to generate a CString from a String")
```

Comment:
Why would getCString fail here? If it did, is there a way to recover instead of asserting?